modules/Kconfig.mcuboot

```diff
@@ -430,4 +430,14 @@ config MCUBOOT_BOOTUTIL_LIB_FOR_DIRECT_XIP
 	  when bootloader is in DirectXIP-revert mode.
 endif
 
+DT_CHOSEN_ZEPHYR_FLASH := zephyr,flash
+
+config MCUBOOT_BOOT_MAX_ALIGN
```

Comment:
this seems a bit problematic, the MCUboot code checks if this is defined - it will always be defined because there is no prompt or anything here, now what happens when a device is uses which does not have a write block size on this, or doesn't have a chosen flash node, does this work or through a CMake/Kconfig configuration error?